contracts/hackatom/src/contract.rs

```diff
@@ -13,6 +13,9 @@ use crate::msg::{
 };
 use crate::state::{State, CONFIG_KEY};
 
+pub const CONTRACT_NAME: &str = env!("CARGO_PKG_NAME");
+pub const CONTRACT_VERSION: &str = env!("CARGO_PKG_VERSION");
```

Comment:
Can't we get those values in the example directly? I.e. without requiring to add those constants to the contract code. It should be the same package.

Reply:
Oh, it does work. Let's do that.